.eslintrc.yml

```diff
@@ -4,11 +4,16 @@ extends:
 parser: babel-eslint
 env:
   browser: true
+settings:
+  import/resolver:
+    node:
+      paths: ['.']
```

Comment:
What is this line about?

Reply:
Eslint was unable to resolve relative path. This line fixes it.